tensorflow/core/kernels/data/iterator_ops.cc

```diff
@@ -145,6 +145,8 @@ class IteratorResource : public ResourceBase {
       params.allocator_getter = [device](AllocatorAttributes attrs) {
         return device->GetAllocator(attrs);
       };
+      params.runner_threadpool_size =
```

Comment:
Is this still needed? The `Params(OpKernelContext*)` constructors should take care of that.

Reply:
Oh, missed this one. Have removed it now.